packages/sdk/test/integration/gc_test.ts

```
@@ -419,32 +731,107 @@ describe('Garbage Collection', function () {
     assert.equal(doc2.getGarbageLen(), 1);
     doc2.update((root) => (root.point.x = 2));
+    assert.equal(
+      versionVectorHelper(doc2.getVersionVector(), [
+        { actor: client1.getID()!, lamport: Long.fromNumber(3) },
+        { actor: client2.getID()!, lamport: Long.fromNumber(5) },
+      ]),
+      true,
+    );
     assert.equal(doc2.getGarbageLen(), 2);
 
     doc1.update((root) => (root.point = { x: 3, y: 3 }));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)